packages/x-components/src/components/currency/base-currency.vue

```diff
@@ -6,11 +6,28 @@
   import Vue from 'vue';
   import { Component, Prop } from 'vue-property-decorator';
   import { currencyFormatter } from '../../utils/currency-formatter';
+  import { XInject } from '../decorators/injection.decorators';
 
   /**
-   * Renders the value received as a property, which always must be a JavaScript number, with the
-   * proper format provided as string property. The rendered tag is a span in order to render a
-   * default inline HTML element.
+   * Renders the value received as a property which always must be a JavaScript number, with the
+   * proper format provided as string property or by injection. The rendered tag is a span in order
+   * to render a default inline HTML element.
+   *
+   * Regarding the format or mask to be defined as string:
+   * - Use 'i' to define integer numbers.
+   * - Use 'd' to define decimal numbers. You can define the length of the decimal part. If the
+   * doesn't include decimals, it is filled with zeros until reach the length defined with 'd's.
+   * - Integer separator must be defined between the 3rd and the 4th integer 'i' of a group.
+   * - Decimal separator must be defined between the last 'i' and the first 'd'. It can be more
+   * than one character.
+   * - If you want to hide the decimal part if it's zero, you can add the `?` symbol after the
+   * decimal separator (e.g. 'i.iii,?dd', for `1234` you would get `1.234` instead of `1.234,00`).
+   * - Set whatever you need around the integers and decimals marks.
+   * - Default mask: 'i.iii,dd' which returns '1.345,67'.
+   *
+   * @remarks The number of 'd', which is the maximum decimal length, MUST match with the length
+   * of decimals provided from the adapter. Otherwise, when the component truncate the decimal
+   * part, delete significant digits.
```

Comment:
`* part, it will delete significant digits.`

Reply:
I put it in present simple.